pyproject.toml

```diff
@@ -64,7 +64,7 @@ name = "Miscellaneous internal changes"
 showcontent = false
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.1"
```

Comment:
a small step for Rasa but a huge for the community. Do we really need this? Does 3.6.0 not work?

Reply:
Just updated it. Now it allows Python 3.6.0, and just uses the old version of `prompt-toolkit` in this case. 
The reason why we need it is that the new version of `prompt-toolkit` relies on `typing.AsyncGenerator` which is not supported in Python 3.6.0. However, the old version (2.x) works perfectly so it can be used with Python 3.6.0.

Reply:
More about why prompt-toolkit doesn't support Python 3.6.0: https://github.com/prompt-toolkit/python-prompt-toolkit/blob/master/setup.py#L32